### PR TITLE
[ui_palette] change folder naming to components

### PR DESCRIPTION
**Pull request by xtrixia**
### Changelog
- Change folder naming `ui_palette` -> `components`
- Update unit test files
- Eslint ignore dist folder
- Prettier ignore dist folder
- Update build-npm

### CODE REVIEW

**Review by Sukmaarifin — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -63,23 +64,23 @@
...
     }
   },
   "scripts": {
-    "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "test-jest": "jest --watch --coverage",
-    "eject": "react-scripts eject",
-    "doc": "npx styleguidist server",
     "build-doc": "npx styleguidist build",
```
Comment (xtrixia) [inline package.json:72]: ini gw re-arrange aja scriptsnya gak ada yg didelete

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+dist/*
```
Comment (royandri) [inline .eslintignore:1]: cukup `dist/` aja gak sih ?

File: package.json (hunk 1 above)
Comment (Sukmaarifin) [reply package.json:72]: okee

File: .eslintignore (hunk 2 above)
Comment (Sukmaarifin) [reply .eslintignore]: yang lainnya juga sih
node_modules
yarn.lock
package-lock.json
dist
public
coverage
src/icons
src/styles

Comment (xtrixia) [reply .eslintignore]: node_modules gak perlu ya, bisa refer ke docs [berikut](https://eslint.org/docs/user-guide/configuring#eslintignore)

**Pull request merged**